draft-ietf-tls-tls13.md

```diff
@@ -5677,7 +5678,8 @@ Prior versions of TLS used the record layer version number for various
 purposes. (TLSPlaintext.legacy_record_version and TLSCiphertext.legacy_record_version)
 As of TLS 1.3, this field is deprecated. The value of
 TLSPlaintext.legacy_record_version MUST be ignored by all implementations.
-The value of TLSCiphertext.legacy_record_version MAY be ignored,
+The value of TLSCiphertext.legacy_record_version is included in the
+additional data for deprotection but MAY otherwise be ignored
 or MAY be validated to match the fixed constant value.
```

Comment:
I guess this works, since just being included in the AAD does not necessarily imply that the sender used the right values.

Reply:
That was my reasoning.